test/abstract/FlowUtilsAbstractTest.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: CAL
+pragma solidity ^0.8.18;
+
+import {Test, Vm} from "forge-std/Test.sol";
+
+struct ERC1155Transfer {
```

Comment:
these transfer structs are already defined in the interface, why are they redefined here rather than imported?